ethereum/core/src/main/java/org/hyperledger/besu/ethereum/core/LogWithMetadata.java

```diff
@@ -101,41 +98,47 @@ public LogWithMetadata(
 
   // The index of this log within the entire ordered list of logs associated with the block this log
   // belongs to.
+  @Override
   public int getLogIndex() {
     return logIndex;
   }
 
+  @Override
   public long getBlockNumber() {
     return blockNumber;
   }
 
+  @Override
   public Hash getBlockHash() {
     return blockHash;
   }
 
+  @Override
   public Hash getTransactionHash() {
     return transactionHash;
   }
 
+  @Override
   public int getTransactionIndex() {
     return transactionIndex;
   }
 
   @Override
   public Address getLogger() {
-    return address;
```

Comment:
Rather than calling super should we just delete it?  Or are there visibility issues that are not readily apparent in this code?

Reply:
Doh! Just an artifact of incremental refactoring making me forget that it's still accessible.